test/e2e/policies.test-suite/topology-aware/n6-hbm-cxl/test04-nrt/code.var.sh

```diff
@@ -0,0 +1,36 @@
+cleanup() {
+    vm-command "kubectl delete pods --all --now"
+    helm-terminate
+}
+
+cleanup
+helm_config=${TEST_DIR}/topology-aware-nrt.cfg helm-launch topology-aware
+
+export get_nrt="kubectl get noderesourcetopologies.topology.node.k8s.io \$(hostname)"
```

Comment:
nit: there's no real need for export here, AFAICT.

Reply:
Thanks for catching that!

I remember putting it there just to make sure it would be visible in `breakpoint` interactive subshell. But forgot to remove it.

Now fixed.